MTA_limit_trends/Comp/run_comp_grad_data_update.py

```diff
@@ -182,6 +190,8 @@ def update_comp_data(gname, msid_list, eyear, etime):
             etime       --- today's date in seconds from 1998.1.1
     output: <data_dir>/<gname>/<msid>_<dtye>_data.fits
     """
+    #make the group subdirectory in case it doesn't exist
+    os.makedirs(f"{DATA_DIR}/{gname}", exist_ok=True)
```

Comment:
Including this command to generate the category directory just in case it does not exists. Useful for creating test output, as well as allowing the script to still run if one of the category sub directories were to accidentally be deleted.